### PR TITLE
DOC: fix a few URLs

### PR DESCRIPTION
**Pull request by partev**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them
- [ ] whatsnew entry

### CODE REVIEW

File: doc/source/getting_started/overview.rst [hunk 1]
```diff
@@ -29,7 +29,7 @@ and :class:`DataFrame` (2-dimensional), handle the vast majority of typical use
 cases in finance, statistics, social science, and many areas of
 engineering. For R users, :class:`DataFrame` provides everything that R's
 ``data.frame`` provides and much more. pandas is built on top of `NumPy
-<https://www.numpy.org>`__ and is intended to integrate well within a scientific
+<https://numpy.org>`__ and is intended to integrate well within a scientific
```
Comment (mzeitlin11) [inline doc/source/getting_started/overview.rst:32]: Any reason for this change, original link seems fine?

File: doc/source/getting_started/overview.rst [hunk 2]
```diff
@@ -168,7 +168,7 @@ The list of the Core Team members and more detailed information can be found on
 Institutional partners
 ----------------------
 
-The information about current institutional partners can be found on `pandas website page <https://pandas.pydata.org/about.html>`__.
+The information about current institutional partners can be found on `pandas website page <https://pandas.pydata.org/about/sponsors.html>`__.
```
Comment (mzeitlin11) [inline doc/source/getting_started/overview.rst:171]: Confirmed this fixes a broken link

File: doc/source/getting_started/overview.rst (hunk 1 above)
Comment (partev) [reply doc/source/getting_started/overview.rst:32]: original link (https://www.numpy.org) redirects to https://numpy.org

**Comment by jreback** [comment]
thanks @partev

**Pull request merged**